graph/typeDefs.graphql

```diff
@@ -140,6 +140,9 @@ input CommentsQuery {
 
   # Sort the results by created_at.
   sort: SORT_ORDER = REVERSE_CHRONOLOGICAL
+
+  # Exclude comments ignored by this user ID
+  notIgnoredBy: String
```

Comment:
I wanted some sort of option in the query so that it could be optional (i.e. a client could still request all comments including the ignored ones if some feature requires it).

But I could still allow for that with this being a boolean property instead of actually having to pass in the User ID. I think I did it this way just to generally make the query explicit or to allow for future changes to allow the query to be an array of user IDs (e.g. if the requesting user was an admin). But it's not obvious we'll ever need that.

Want me to change it? e.g. to an `excludeIgnored` boolean?